src/adapters/suggest.js

```diff
@@ -76,17 +76,20 @@ export default class Suggest {
         let remotes = []
         let categories = []
         pois.forEach((poi) => {
-          if (poi instanceof PoiStore) {
+          if (poi instanceof PoiStore && favorites.length < 2) {
+            // 2 favorites pois max
             favorites.push(poi)
```

Comment:
With this implementation, extra favorites (if length >2) will be pushed to `remotes`.  
For example on a empty search bar, all favorites are returned, but only 2 are described as "favorites":
![image](https://user-images.githubusercontent.com/4726554/58887652-4df4ce00-86e6-11e9-9136-c10f0d1bf045.png)

Actually we may want to display more than 2 favorites in that case (think about choosing a favorite address when selecting an origin or a destination on the directions panel)

Reply:
I made some updates in order to avoid getting favorites into remotes.
I got some new guidelines like :
- 5 favorites max will be displayed on empty query
- when query is not empty :
display 7 items maximum
2 favorites maximum
1 category maximum
remaining items has to be filled with remotes
- currently working on displaying addresses on second-line's item